packages/galaxy/setup.cfg

```diff
@@ -0,0 +1,261 @@
+[metadata]
+author = Galaxy Project and Community
+author_email = galaxy-committers@lists.galaxyproject.org
+classifiers =
+    Development Status :: 5 - Production/Stable
+    Environment :: Console
+    Intended Audience :: Developers
+    License :: OSI Approved :: Academic Free License (AFL)
+    Natural Language :: English
+    Operating System :: POSIX
+    Programming Language :: Python :: 3
+    Programming Language :: Python :: 3.8
+    Programming Language :: Python :: 3.9
+    Programming Language :: Python :: 3.10
+    Programming Language :: Python :: 3.11
+    Programming Language :: Python :: 3.12
+    Topic :: Software Development
+    Topic :: Software Development :: Code Generators
+    Topic :: Software Development :: Testing
+description = Galaxy
+keywords =
+    Galaxy
+license = AFL
+license_files =
+    LICENSE
+long_description = file: README.rst, HISTORY.rst
+long_description_content_type = text/x-rst
+name = galaxy
+url = https://github.com/galaxyproject/galaxy
+version = 24.1.2
+
+[options]
+include_package_data = True
+install_requires =
```

Comment:
Are there tools to keep this up to date ?

Reply:
No, I guess it should be part of the package release process?

Reply:
Yep, this could be handled as part of `create_point_release` in https://github.com/galaxyproject/galaxy-release-util/blob/main/galaxy_release_util/point_release.py#L597
Is this is a subset of `pinned-requirements.txt`? (I assume it's the difference between that and the union of requirements for all other packages?)

Reply:
:point_up: and I can add that to galaxy-release-util unless you'd like to handle it.

Reply:
https://github.com/galaxyproject/galaxy-release-util/pull/23